[2023-11-24 22:43:52,873] ERROR log_exception() L1449 Exception on /raa [GET] - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py L1449
Traceback (most recent call last):
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_openapi3/scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_openapi3/do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "/Users/tairik/PycharmProjects/mvp-tairik/rest_api_pets/app.py", line 76, in get_raa
    return apresenta_raa(prediction), 200
  File "/Users/tairik/PycharmProjects/mvp-tairik/rest_api_pets/schemas/dog.py", line 38, in apresenta_raa
    "id": dog.id,
AttributeError: 'numpy.ndarray' object has no attribute 'id'
[2023-11-24 22:43:52,909] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:43:52] "[35m[1mGET /raa HTTP/1.1[0m" 500 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:46:10,537] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:46:13,649] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:46:13] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:46:13,834] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:46:13] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:46:13,851] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:46:13] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:46:13,862] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:46:13] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:46:14,676] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:46:14] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:46:14,823] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:46:14] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:46:22,801] ERROR log_exception() L1449 Exception on /raa [GET] - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py L1449
Traceback (most recent call last):
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_openapi3/scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
  File "/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/flask_openapi3/do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
  File "/Users/tairik/PycharmProjects/mvp-tairik/rest_api_pets/app.py", line 61, in get_raa
    return apresenta_raa(prediction), 200
  File "/Users/tairik/PycharmProjects/mvp-tairik/rest_api_pets/schemas/dog.py", line 38, in apresenta_raa
    "id": dog.id,
AttributeError: 'numpy.ndarray' object has no attribute 'id'
[2023-11-24 22:46:22,834] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:46:22] "[35m[1mGET /raa HTTP/1.1[0m" 500 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:47:36,971] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:47:37,370] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:47:37] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:47:37,438] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:47:37] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:47:37,519] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:47:37] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:47:37,523] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:47:37] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:47:38,117] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:47:38] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:47:38,908] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:47:38] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:48:40,658] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:4001
 * Running on http://192.168.68.102:4001 (Press CTRL+C to quit) - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:48:42,124] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:48:42] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:48:42,201] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:48:42] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:48:42,205] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:48:42] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:48:42,210] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:48:42] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:48:42,752] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:48:42] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
[2023-11-24 22:48:42,940] INFO _log() L224  192.168.68.102 - - [24/Nov/2023 22:48:42] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/Users/tairik/PycharmProjects/mvp-tairik/venv/lib/python3.9/site-packages/werkzeug/_internal.py L224 
